tobac/tests/test_utils.py

```diff
@@ -55,3 +56,27 @@ def test_spectral_filtering():
         )
         >= 1
     )
+
+
+def test_combine_tobac_feats():
+    """tests tobac.utils.combine_tobac_feats
+    Test by generating two single feature dataframes,
+    combining them with this function, and then
+    testing to see if a single dataframe
+    matches.
+    """
+    import datetime
+    import pandas as pd
+
+    single_feat_1 = tb_test.generate_single_feature(
+        0, 0, start_date=datetime.datetime(2022, 1, 1, 0, 0), frame_start=0
+    )
+    single_feat_2 = tb_test.generate_single_feature(
+        1, 1, start_date=datetime.datetime(2022, 1, 1, 0, 5), frame_start=0
+    )
+
+    combined_feat = tb_utils.combine_tobac_feats([single_feat_1, single_feat_2])
+
+    tot_feat = tb_test.generate_single_feature(0, 0, num_frames=2, frame_start=0)
```

Comment:
It might be reasonable to make `spd_h1` and `spd_h2` explicit here, just to be sure it is obvious why these are the same features and in case the default values ever change.

Reply:
Good point. Updated.